assets/dev/js/editor/regions/navigator/elements/commands/toggle-folding.js

```diff
@@ -2,31 +2,23 @@ import CommandNavigator from './base/command-navigator';
 
 export class ToggleFolding extends CommandNavigator {
 	apply( args ) {
-		const { containers = [ args.container ], callback, state } = args;
+		const { containers = [ args.container ], state } = args;
 
 		containers.forEach( ( container ) => {
-			if ( ! container.children.length ) {
-				return;
-			}
-
-			const view = container.args.navigatorView,
-				isActive = view.ui.item.hasClass( 'elementor-active' );
+			const isActive = ( ( _state ) => _state[ 'navigator/folding' ][ container.id ] )(
+				$e.store.getState()
+			);
 
 			if ( isActive === state ) {
 				return;
 			}
 
-			const newArgs = { container };
-
-			if ( callback ) {
```

Comment:
What is the `callback`?

Reply:
Functionality that @iNewLegend added to be compatible with the old UI (Marionette, jQuery). With React, it's not needed anymore.